BAZProject/Common/Networking/Models/ReviewCollectionDataObject.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ReviewCollectionDataObject.swift
+//  BAZProject
+//
+//  Created by gescarcega on 05/05/23.
+//
+
+import Foundation
+
+struct ReviewCollectionDataOject: Decodable, Equatable, Hashable {
```

Comment:
Hay un pequeño typo.
```suggestion
struct ReviewCollectionDataObject: Decodable, Equatable, Hashable {
```